.github/workflows/lint.yml

```diff
@@ -59,35 +60,6 @@ jobs:
             path: crates/sallyport/Cargo.toml
             target: --target=x86_64-unknown-none
 
-  clippy-single-backends:
```

Comment:
Why remove the clippy job?
I would expect that we'd just use `cargo clippy --workspace`